jupyter_core/command.py

```diff
@@ -49,6 +49,7 @@ def list_subcommands():
     Returns a list of jupyter's subcommand names, without the `jupyter-` prefix.
     Nested children (e.g. jupyter-sub-subsub) are not included.
```

Comment:
Rearranging `install-nbextension` into an `nbextension` subcommand seems neatest (getting rid of this PR)